### PR TITLE
[Keyboard] Relocalize and Update p1800fl

### PR DESCRIPTION
**Pull request by team0110**
## Description

We are currently in Production Stage, so i could like to relocalize p1800fl to desired folder, also update VIA keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/team0110/p1800fl/rules.mk [hunk 1]
```diff
@@ -9,22 +9,22 @@ MCU = atmega32u4
...
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes		# Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite		# Virtual DIP switch configuration
 MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes		# Audio control and System control
-CONSOLE_ENABLE = yes			# Console for debug
-COMMAND_ENABLE = yes			# Commands for debug and configuration
+CONSOLE_ENABLE = no			# Console for debug
+COMMAND_ENABLE = no			# Commands for debug and configuration
```
Comment (fauxpark) [inline keyboards/team0110/p1800fl/rules.mk:21]:
```suggestion
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
```

File: keyboards/team0110/p1800fl/rules.mk [hunk 2]
```diff
@@ -9,22 +9,22 @@ MCU = atmega32u4
...
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = yes			# USB Nkey Rollover
 BACKLIGHT_ENABLE = yes		# Enable keyboard backlight functionality
```
Comment (fauxpark) [inline keyboards/team0110/p1800fl/rules.mk:26]:
```suggestion
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
```

File: keyboards/team0110/p1800fl/rules.mk [hunk 3]
```diff
@@ -9,22 +9,22 @@ MCU = atmega32u4
...
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 MIDI_ENABLE = no            # MIDI support
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (fauxpark) [inline keyboards/team0110/p1800fl/rules.mk:30]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/team0110/p1800fl/rules.mk [hunk 4]
```diff
@@ -9,22 +9,22 @@ MCU = atmega32u4
 #   QMK DFU      qmk-dfu
 #   ATmega32A    bootloadHID
 #   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/team0110/p1800fl/rules.mk:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by team0110** [comment]
Thank for your advise @fauxpark

File: keyboards/team0110/p1800fl/rules.mk [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/team0110/p1800fl/rules.mk:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Since this is moving the board's location, it should target the develop branch.

**Comment by marhalloweenvt** [comment]
> Since this is moving the board's location, it should target the develop branch.

Thank for your reply.
Does it mean i need to commit it in Dev Bracnh? And how long will it take to be merged in master branch?

**Comment by drashna** [comment]
> > Since this is moving the board's location, it should target the develop branch.
> 
> Thank for your reply, @drashna .
> Does it mean i need to commit it in Dev Bracnh? And how long will it take to be merged in master branch?

It would be targeted to and merged into `develop`, yes.  And the timeline is:
https://docs.qmk.fm/#/breaking_changes?id=important-dates

So a month before it hit master.

**Review by tzarc — APPROVED** [review]

**Comment by tzarc** [comment]
Apparently there's now a conflict.

**Comment by marhalloweenvt** [comment]
> Apparently there's now a conflict.

How can I help you to resolve the conflict? From my side, I can't see any recommends.

**Comment by drashna** [comment]
> > Apparently there's now a conflict.
> 
> How can I help you to resolve the conflict? From my side, I can't see any recommends.

You'd need to rebase the branch, I think.

**Comment by drashna** [comment]
Thanks!

And I fixed up the merge conflicts for you.  
Mostly, it's just `git merge upstream/develop` and fix the conflicts.  (that or `git rebase upstream/develop`)

**Comment by marhalloweenvt** [comment]
> Thanks!
> 
> And I fixed up the merge conflicts for you.
> Mostly, it's just `git merge upstream/develop` and fix the conflicts. (that or `git rebase upstream/develop`)

Thanks for your kind support. If there is any issues, please let me know.

**Pull request merged**